src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -101,7 +101,7 @@ void GcsResourceManager::HandleDeleteResources(
     const rpc::DeleteResourcesRequest &request, rpc::DeleteResourcesReply *reply,
     rpc::SendReplyCallback send_reply_callback) {
   NodeID node_id = NodeID::FromBinary(request.node_id());
-  RAY_LOG(DEBUG) << "Deleting node resources, node id = " << node_id;
+  RAY_LOG(INFO) << "Deleting node resources, node id = " << node_id;
```

Comment:
I don't think we use this endpoint anymore. Isn't it for dymanic resources? It is deprecated.